Rakefile

```diff
@@ -0,0 +1,38 @@
+class String
+  def shuffle
+    split('').shuffle.join
+  end
+
+  def shuffle!
+    replace shuffle
+  end
+end
+
+msg = "jekyll-namesake"
+
+task :default do
+  msg = "#{msg}\nFeel free to run a task for more instructions."
+  system "echo #{msg}"
+  system "rake --tasks"
+end
+
+desc 'Generate an encoded email address to stop spammers'
```

Comment:
What's this all about?

Reply:
It generates an encoded email, which is decoded with javascript into the "hear from you" hyperlink. See the readme for destructions.